### ISSUE TITLE
SOMEONE PLEASE HELP ME

### ISSUE THREAD

**Issue by Githuber1-ai** [haveno-dex/haveno#1243]
LIKE I WROTE IN THE PREVIOUS ISSUE I CANT FUCKING LOG IN THIS SHITTY HAVENO APP BECAUSE IT SAYS THIS 

![5D19FAE5-50AC-42CF-BE50-125C98F9E9D5](https://github.com/user-attachments/assets/c68a51c9-1c2e-4894-94df-28c9180184ae)

I LITERALLY TRIED EVERYTHING HOW THE FUCK DO I LOG IN I HAVE XMR IN THAT. THIS IS BS. SOMEONE TELL ME HOW TO FIX THIS. I DEMAND A REFUND FOR ALL MY XMR LOST IF THIS ISNT FIXED.

WORST APP EVER!!!! NEVER EVER USE EVER!!!

PLEASE PLEASE PLEASE PLEASE SOMEONE HELP ME ILL GIVE YOU SOME MONERO AS A RETURN IF YOUR HELP. I USE WINDOWS 10 IF THIS HELPS.

**Comment by charcharchar** [comment]
shutdown and restart. make a backup of the haveno files in your user directly as stated in previous issue by using file explorer. try installing on a different computer and move your copied files in this user directly. im not sure how to dumb it down any more, consult a friend or google if you cant find out how to back up the files.

**Comment by Githuber1-ai** [comment]
> Duplicate of #1242
> 
> Follow these instructions and report your results: [#1242 (comment)](https://github.com/haveno-dex/haveno/issues/1242#issuecomment-2326165804)

I downloaded the update. It works now. Thank you so much. Please give me your Monero address so I can give you some xmr as a reward.
